### PR TITLE
mwc-select -> ha-select

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Replace `mwc-select` with `ha-select`

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-form/ha-form-multi_select.ts [hunk 1]
```diff
@@ -18,6 +17,7 @@ import type { HaCheckbox } from "../ha-checkbox";
 import "../ha-formfield";
 import "../ha-svg-icon";
 import "../ha-textfield";
+import "../ha-select";
```
Comment (zsarnett) [inline src/components/ha-form/ha-form-multi_select.ts:20]: Is this used?

Comment (zsarnett) [reply src/components/ha-form/ha-form-multi_select.ts:20]: nvm

Comment (zsarnett) [inline src/components/ha-form/ha-form-multi_select.ts:20]: Is this used?

File: src/components/ha-qr-scanner.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import "@material/mwc-button/mwc-button";
 import "@material/mwc-list/mwc-list-item";
-import "@material/mwc-select/mwc-select";
+import "./ha-select";
```
Comment (zsarnett) [inline src/components/ha-qr-scanner.ts:3]: Used?

File: src/dialogs/more-info/controls/more-info-humidifier.ts [hunk 3]
```diff
@@ -170,7 +170,7 @@ class MoreInfoHumidifier extends LitElement {
         color: var(--primary-text-color);
       }
 
-      mwc-select {
+      ha-select {
         width: 100%;
       }
```
Comment (zsarnett) [inline src/dialogs/more-info/controls/more-info-humidifier.ts:175]:
```suggestion
```

**Review by zsarnett — DISMISSED** [review]

File: src/dialogs/more-info/controls/more-info-media_player.ts [hunk 4]
```diff
@@ -135,12 +135,9 @@ class MoreInfoMediaPlayer extends LitElement {
...
         ? html`
             <div class="source-input">
-              <ha-svg-icon
-                class="source-input"
-                .path=${mdiLoginVariant}
-              ></ha-svg-icon>
-              <mwc-select
+              <ha-select
                 .label=${this.hass.localize("ui.card.media_player.source")}
+                .icon=${true}
```
Comment (balloob) [inline src/dialogs/more-info/controls/more-info-media_player.ts:140]: Would this work? 
```suggestion
                icon
```

**Review by balloob — DISMISSED** [review]

File: src/components/ha-select.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { SelectBase } from "@material/mwc-select/mwc-select-base";
+import { styles } from "@material/mwc-select/mwc-select.css";
+import { html, nothing } from "lit";
+import { customElement } from "lit/decorators";
+
+@customElement("ha-select")
+export class HaSelect extends SelectBase {
```
Comment (balloob) [inline src/components/ha-select.ts:7]: Should this override the icon type to be `boolean` so an attribute would work?

**Review by zsarnett — DISMISSED** [review]

**Review by zsarnett — APPROVED** [review]

**Pull request merged**